packages/react-components/react-popover/stories/Popover/PopoverAnchorToCustomTarget.stories.tsx

```diff
@@ -41,7 +41,7 @@ export const AnchorToCustomTarget = () => {
           <Button>Popover trigger</Button>
         </PopoverTrigger>
 
-        <PopoverSurface>
+        <PopoverSurface tabIndex={-1}>
```

Comment:
This looks as good change to me. 

But when I tried with NVDA, it narrates the popover as "clickable" (what is not correct in this case):
![image](https://github.com/microsoft/fluentui/assets/22620150/95c17e5b-326e-4308-91ef-7817d3931b8e)

what I saw in the past, it is some "feature" of NVDA it narrates "clickable" when there is "onclick" listener on the element...
I think we shouldn't have the "onclick" listener on not actionable element... 

this "clickable" narrated by NVDA can be pretty confusing for the user...

Reply:
Let's create an issue for this separately to track

Reply:
This is an NVDA thing that is widespread -- an NVDA user would almost certainly be so used to this that it wouldn't be an issue